src/client/client.c

```diff
@@ -1,9 +1,9 @@
-#include "client.h"
 #include "libhirte/bus/utils.h"
-#include <stdio.h>
+
+#include "client.h"
 
 int list_units_on_all(UNUSED Client *client) {
-        printf("Listing units on all\n");
+        printf("Listing units on all");
```

Comment:
missing the trailing "\n" ;)

(and on a few below, cf the diff here)

Reply:
Changed.